src/System.Private.ServiceModel/src/System.Private.ServiceModel.csproj

```diff
@@ -1,5 +1,8 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="12.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003" DefaultTargets="Build">
+  <PropertyGroup>
+    <Configuration Condition="'$(Configuration)'==''">Windows_Debug</Configuration>
+  </PropertyGroup>
```

Comment:
I buddy tested just this single edit separately a few days ago on my 2 machines that both failed some OuterLoops with the basic build.cmd /p:WithCategories=OuterLoop.  This change appears to have eliminated the race condition that caused that failure, and it allowed both machines to pass.